### PR TITLE
[Styles] Add survey point & shelter icon

### PR DESCRIPTION
**Pull request by endim8**
closes https://github.com/organicmaps/organicmaps/issues/2181

Adds day and night survey point & shelter icons, also adds rule to render leisure=bird_hide with shelter icon, and fixes a small typo.

Requires map regeneration.

Old and new shelter icon
vsfsfsfs
shelter at night:
Screenshot 2022-03-20 at 00 00 57

survey_point:
156080161-d9d2a37a-4efa-4bfa-b75f-815e84ea4d9e
night:
Screenshot 2022-03-20 at 00 09 05

Signed-off-by: Harry Bond <endim8@pm.me>

### CODE REVIEW

**Review by pastk — CHANGES_REQUESTED** [review]

**Comment by pastk** [comment]
Could you add screenshots of new icons in dark mode please?

File: data/styles/clear/include/Icons.mapcss [hunk 1]
```diff
@@ -220,6 +220,8 @@ node[leisure=picnic_table],
 area[leisure=picnic_table],
 node[amenity=shelter],
 area[amenity=shelter],
+node[leisure=bird_hide],
+area[leisure=bird_hide],
```
[the remark this answers is not in the document]
Comment (endim8) [reply data/styles/clear/include/Icons.mapcss:224]: ah oops, I'll add it in a separate PR 👍

I need to do the additional things for survey_point too, right?

File: data/styles/clear/include/Icons.mapcss [hunk 2]
```diff
@@ -2055,8 +2063,9 @@ node|z18-[amenity=bbq],
 area|z18-[amenity=bbq]
 {icon-image: bbq-m.svg;font-size: 11;-x-me-text-priority:16000;text-optional: true;}
 
-node|z19-[man_made=survey_point]
-{icon-image: survey_point-m.svg;}
+node|z17-[man_made=survey_point],
+area|z17-[man_made=survey_point]
+{icon-image: survey_point-m.svg;font-size: 11;text-optional: true;}
```
[the remark this answers is not in the document]
Comment (endim8) [reply data/styles/clear/include/Icons.mapcss:2065]: i think i've added this correctly now, i'm having issues generating a mwm to test it though

File: data/styles/clear/include/Icons.mapcss (hunk 1 above)
[the remark this answers is not in the document]
Comment (endim8) [reply data/styles/clear/include/Icons.mapcss]: > Could you add screenshots of new icons in dark mode please?

I've added one for the shelter, but I'm having difficulty generating a mwm for the survey_point. It should work fine though, it has a dark mode icon with the same opacity and colour as all the others :)

**Comment by endim8** [comment]
how does it look? :) i think it should be correct now

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -11051,6 +11051,12 @@
     uk = Просіка
     zh-Hans = 树林分界线
 
+  [type.man_made.survey_point]
+    en = Survey Point
+    de = Vermessungspunkt
+    fr = Point d'enquête
+    ru = Точка наблюдения
```
Comment (pastk) [inline data/strings/types_strings.txt:11058]: ru = Геодезический пункт
be = Геадэзічны пункт
uk = Геодезичний пункт

File: data/categories.txt [hunk 4]
```diff
@@ -7149,6 +7149,12 @@ el:Φάρος
 sk:Maják
 fa:فانوس دریایی
 
+man_made-survey_point
+en:4survey|4survey point|4trigonometry
+ru:Точка наблюдения
+de:Vermessungspunkt
+fr:Point d'enquête
+
```
Comment (pastk) [inline data/categories.txt:7157]: en:4Survey Point|survey marker|survey benchmark|4geodetic mark|geodetic vertex|4triangulation station|4trigonometrical point|trig point|trig pillar|4trig station|trig beacon|trig
ru:4Геодезический пункт|Геодезический знак|4тригонометрический пункт|пункт триангуляции|тригопункт|4триангулятор
be:4Геадэзічны пункт
uk:4Геодезичний пункт

For technical terms automatic translation is often misleading, its better to take translations from e.g. linked Wikipedia articles. Translated OSM wiki pages are useful too (unless they were translated automatically).

E.g. a french "Point d'enquête" you suggested looks like auto translation, this term is not mentioned neither in French [OSM wiki](https://wiki.openstreetmap.org/wiki/FR:Tag:man_made%3Dsurvey_point), nor in wikipedia [1](https://fr.wikipedia.org/wiki/Borne_g%C3%A9od%C3%A9sique) [2](https://fr.wikipedia.org/wiki/Poste_de_triangulation)

File: data/styles/clear/include/Icons.mapcss [hunk 5]
```diff
@@ -68,6 +68,7 @@ node[amenity=waste_basket],
 node[amenity=waste_disposal],
 area[amenity=waste_disposal],
 node[man_made=survey_point],
+area[man_made=survey_point],
```
Comment (pastk) [inline data/styles/clear/include/Icons.mapcss:71]: according to OSM wiki it can't be an area, only a node

**Review by pastk — CHANGES_REQUESTED** [review]
Its getting better!
Night screenshots look good to me!

File: data/categories.txt (hunk 4 above)
Comment (endim8) [reply data/categories.txt:7163]: fixed, translations taken from https://www.wikidata.org/wiki/Q352956 👍

File: data/styles/clear/include/Icons.mapcss (hunk 5 above)
Comment (endim8) [reply data/styles/clear/include/Icons.mapcss]: fixed

File: data/strings/types_strings.txt (hunk 3 above)
Comment (endim8) [reply data/strings/types_strings.txt]: added, thank you 👍

**Comment by endim8** [comment]
i've updated it all, it should be good now :)

**Review by pastk — APPROVED** [review]
Looks good to me, thanks!

Test failures look unrelated.

**Comment by biodranik** [comment]
> Looks good to me, thanks!
> 
> Test failures look unrelated.

They ARE related:

```
CHECK(langCode != kUnsupportedLocaleCode) Invalid language code: et at line: 7157
```

File: data/categories.txt [hunk 6]
```diff
@@ -7149,6 +7149,18 @@ el:Φάρος
 sk:Maják
 fa:فانوس دریایی
 
+man_made-survey_point
+en:4Survey Point|survey marker|survey benchmark|4geodetic mark|geodetic vertex|4triangulation station|4trigonometrical point|trig point|trig pillar|4trig station|trig beacon|trig
+ru:4Геодезический пункт|Геодезический знак|4тригонометрический пункт|пункт триангуляции|тригопункт|4триангулятор
+de:4Vermessungspunkt
+el:4Γεωδαιτικό σημείο
+et:4Geodeetiline märk
```
Comment (biodranik) [inline data/categories.txt:7157]: Estonian language should be added in the code to work properly. I would recommend doing it separately in another PR.

Comment (endim8) [reply data/categories.txt]: ah thank you! i've removed it now, hopefully there are no more issues :)

**Comment by endim8** [comment]
oh damnit hang on, i've accidentally rebased onto pastk-strings 🤦‍♂️ one sec

File: data/categories.txt [hunk 7]
```diff
@@ -7149,6 +7149,17 @@ el:Φάρος
...
 
+man_made-survey_point
+en:4Survey Point|survey marker|survey benchmark|4geodetic mark|geodetic vertex|4triangulation station|4trigonometrical point|trig point|trig pillar|4trig station|trig beacon|trig
+ru:4Геодезический пункт|Геодезический знак|4тригонометрический пункт|пункт триангуляции|тригопункт|4триангулятор
+de:4Vermessungspunkt
+el:4Γεωδαιτικό σημείο
+fr:4Borne géodésique
+ja:測量標
+pl:4Znak geodezyjny
+be:4Геадэзічны пункт
```
Comment (biodranik) [inline data/categories.txt:7160]: nit: sort languages by alphabet, except en which always goes first.

**Review by biodranik — APPROVED** [review]

File: data/categories.txt (hunk 7 above)
Comment (endim8) [reply data/categories.txt]: fixed 👍

**Comment by endim8** [comment]
This should be good to merge now finally :)

**Review by biodranik — APPROVED** [review]

**Pull request merged**